newswires/client/src/App.tsx

```diff
@@ -74,9 +75,26 @@ export function App() {
 						</p>
 					</EuiToast>
 				)}
+				{isRestricted(query.dateRange?.end) && (
```

Comment:
if both toasts are rendered, one will sit on top of the other: 
<img width="806" alt="image" src="https://github.com/user-attachments/assets/561b62a5-6910-406d-b636-21b20724302f" />

(I adjusted the opacity of the top one in dev tools for that screenshot)

Reply:
one option here might be to automatically disable 'auto-refresh' if `isRestricted`, and display the note about not updating next to the auto refresh toggle?

probably only worth doing that if it's fairly quick and easy to implement though, as I suspect we might end up tweaking refresh behaviour before too long...

Reply:
I tried to automatically disable 'auto-refresh' but it got to messy. I will prioritise the offline message.

Reply:
> I tried to automatically disable 'auto-refresh' but it got to messy

got it 👍 I was looking at refactoring the auto refresh state in a separate branch anyway, so if I get round to that I'll see if it'd make it any more straightforward to implement that later